epm/pkg/utils/cmsg.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import(
+	"net"
+	"syscall"
+)
+
+func RecvFd(socksPath string) (int, error) {
```

Comment:
Plz check whether https://github.com/opencontainers/runc/blob/master/libcontainer/utils/cmsg.go#L41 can be reused partially in your implementation.

Reply:
ok, I will check it. Thanks